checkrebuild

```diff
@@ -22,17 +22,26 @@ get_unofficial_pkgs() {
     comm -23 <(pacman -Qq | sort) <(pacman -Sql "${repos_to_skip[@]}" 2>/dev/null | sort)
 }
 
-filter_executable() {
-    while read -r file; do
-        [[ -f "$file" && -x "$file" ]] && echo "$file"
-    done
+newline2null(){
+    perl -pe 's/\n/\0/'
 }
 
 get_package_files() {
     xargs pacman -Qql |
-    perl -pe 's/\n/\0/' |
-    xargs -0 readlink -f |
-    filter_executable
+    newline2null |
+    xargs -0 readlink -e |
+    sort -u
+}
+
+filter_executable() {
+    newline2null |
+    LANG=C xargs -0 stat --printf "%F %a\t%n\n" |
+    grep -P "^regular file [^64]" |
+    cut -f2- |
+    newline2null |
+    xargs -0 file |
+    grep -E "ELF (64|32)-bit" |
```

Comment:
Could this be just grepping for ` ELF ` without specifying for bit-ness? Or there are more ELF files that we want to ignore? Or it's all the same? 😄

Reply:
No, it's just to prevent us from ignoring, rare case of a file that contains literal 'ELF' in they path. :godmode:

Reply:
Ok, found a way to get around this limitation, also dropping `cut -d':'`